src/App.js

```diff
@@ -1,28 +1,8 @@
-import logo from './logo.svg';
```

Comment:
- [ ] Please delete this logo too from the src folder as it's no longer in use and it was part of the boilerplate that CRA creates.

Reply:
Thank you for that observation. I have deleted this svg